coriolis/osmorphing/osmount/base.py

```diff
@@ -107,12 +108,25 @@ def _get_pvs(self):
             if line == "":
                 continue
             line = line.strip().split(":")
+            if len(line) < PVDISPLAY_VALUES_COUNT:
+                continue
             if pvs.get(line[1]) is None:
                 pvs[line[1]] = [line[0], ]
             else:
                 pvs[line[1]].append(line[0])
         return pvs
 
+    def _check_vgs(self):
+        try:
+            self._exec_cmd("sudo vgck")
+        except Exception as ex:
+            raise exception.CoriolisException(
+                "An LVM-related problem has been encountered which prevents "
+                "the OSMorphing from proceeding further. Please ensure that "
+                "the source VM's LVM configuration is correct and the VM is "
+                "able to use all LVM volumes on the source. Error occured "
+                "while scanning LVM PVs: %s" % str(ex))
```

Comment:
This is not technically correct as you're running `vgck`.
As per `man vgck`, please change this to `"Error occured while checking the consistency of all LVM volume groups."`